woocommerce/woocommerce
# Fix woocommerce-admin "packages" commands

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes the following WCA commands

- packages:build
- packages:watch
- run:packages

The current commands are not working because the `--filter` value is incorrect.

> $ pnpm run --filter ../../packages/js/
> No projects matched the filters in "/Users/chihsuan/Projects/woocommerce"



### How to test the changes in this Pull Request:




1. Run `cd ./plugins/woocommerce-admin`
2. Run `pnpm packages:build`
3. Confirm it builds all packages
4. Run `pnpm packages:watch`
5. Change any JS file in @woocomemrce/components or any watching packages
6. Confirm it rebuilds the file.



### Other information:




### FOR PR REVIEWER ONLY:
